web/app/js/components/GrafanaLink.jsx

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 
 export default class GrafanaLink extends React.Component {
   render() {
+    let resource = this.props.resource.toLowerCase().replace(" ", "-");
```

Comment:
this generates a link with
`var-replication-controller=authors`
...but i think it should be:
`var-replication_controller=authors`

Reply:
ooops yes, you're right!

Reply:
wait, I realized it's actually `conduit-replication-controller` for the dashboard name and `replication_controller` for the variable...